config/locales/mailers/en.yml

```diff
@@ -15,3 +15,11 @@ en:
       button_text: 'View Results'
       explicit_link: "If the above button doesn't work, copy-and-paste this link into your web browser: %{link}"
       disclaimer: 'This is an automated email. Please do not reply.'
+
+    grouping_invite_email:
+      subject: 'MarkUs Notification (%{course}) You have been invited to a group!'
+      greeting: 'Hello %{first_name} %{last_name},'
+      text_body: '%{first_name} %{last_name} has invited you to join a group for %{assignment}.'
```

Comment:
Please add a second sentence here, "If you wish to accept this invitation, you need to do so *before* the assignment due date."